boa3/builtin/interop/runtime/__init__.py

```diff
@@ -54,6 +54,7 @@ def get_trigger() -> TriggerType:
 get_platform: str = ''
 invocation_counter: int = 0
 entry_script_hash: UInt160 = UInt160()
+get_script_container: Any = None
```

Comment:
if this is a property, it's better to name it simply as `script_container`
if you want to keep `get` in the name, change it to a function instead